src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpChannelFactory.cs

```diff
@@ -273,7 +273,11 @@ private HttpCookieContainerManager GetHttpCookieContainerManager()
                     var clientHandler = GetHttpMessageHandler(to, clientCertificateToken);
                     clientHandler.AutomaticDecompression = DecompressionMethods.Deflate | DecompressionMethods.GZip;
                     clientHandler.UseCookies = _allowCookies;
-                    clientHandler.CookieContainer = _allowCookies ? _httpCookieContainerManager.CookieContainer : null;
+                    if (_allowCookies)
+                    {
+                        clientHandler.CookieContainer = _httpCookieContainerManager.CookieContainer;
```

Comment:
This will still break actual cookie usage in UWP apps. We need to fix this more generally rather than avoid triggering the issue in the general case. I'm okay with this as a short term fix.